workspace.jsonc

```diff
@@ -282,7 +282,7 @@
         "parallel-webpack": "2.6.0",
         "parse-package-name": "0.1.0",
         "path-to-regexp": "6.2.0",
-        "penpal": "5.3.0",
+        "penpal": "6.2.1",
```

Comment:
did we check that this update doesnt break stuff?

Reply:
yes I did